sources/packages/api/src/services/student/student.service.ts

```diff
@@ -261,13 +261,13 @@ export class StudentService extends RecordDataModelService<Student> {
     lastName: string,
     appNumber: string,
   ): Promise<Student[]> {
-    const applicationExistsQuery = this.applicationRepo
-      .createQueryBuilder("application")
-      .where("application.applicationNumber Ilike :appNumber")
-      .andWhere("application.applicationStatus != :overwrittenStatus")
-      .select("1");
     const searchQuery = this.repo
       .createQueryBuilder("student")
+      .leftJoinAndSelect(
```

Comment:
do we need all Application objects here, Maybe you can retrieve the needed columns

Reply:
Feedback comment addressed.